### PR TITLE
(SIMP-3961) core: Add docker modules

### PR DESCRIPTION
**Pull request by jeefberkey**
- Add simp/simp_docker to Puppetfile.tracking
- Add puppetlabs/docke to Puppetfile.tracking
- Bump simp.spec version to 6.1.1-RC1

SIMP-3961 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
This should wait until the simp_docker profile is good to go.

File: build/rpm/dependencies.yaml [hunk 1]
```diff
@@ -61,6 +61,12 @@
 'augeasproviders_postgresql':
   :release: '2016'
 
+'docker':
+  :requires:
+    # exclude pupmod-stahnma-epel
```
Comment (lnemsick-simp) [inline build/rpm/dependencies.yaml]: Will this cause a problem?  Is it possible for the user to need epel-related functionality from the docker module in a SIMP system?

Comment (jeefberkey) [reply build/rpm/dependencies.yaml]: https://github.com/puppetlabs/puppetlabs-docker/issues/9

I think this feature will be removed and I don't personally agree with epel being managed by this module myself.

It is possible for a user to have missing functionality without this module

**Comment by lnemsick-simp** [comment]
simp.spec should be updated to pick up the new component rpms

**Pull request merged**